django_filters/filters.py

```diff
@@ -217,26 +217,26 @@ def is_noop(self, qs, value):
         return False
 
     def filter(self, qs, value):
-        value = value or ()  # Make sure we have an iterable
-
-        if self.is_noop(qs, value):
+        if not value:
```

Comment:
This seems sensible to me, however the standard filter methods uses `if value in ([], (), {}, None, '')`. I've always wondered why this was the case instead of just `if not value`.